src/main/scala/com/thenewmotion/chargenetwork/ocpp/charger/ChargerApp.scala

```diff
@@ -33,10 +38,26 @@ object ChargerApp {
 
     val url = new URI(config.chargeServerUrl())
     val charger = if (connectionType == Json) {
-      new OcppJsonCharger(config.chargerId(), config.numberOfConnectors(), url)
+      new OcppJsonCharger(
+        config.chargerId(),
+        config.numberOfConnectors(),
+        url,
+        config.authPassword.get
+      )(config.keystoreFile.get.fold(SSLContext.getDefault) { keystoreFile =>
+        SslContext(
+          keystoreFile,
+          config.keystorePassword()
```

Comment:
the apply method of scalops internally does a get which is safe to do because we always supply a default.